share/functions/fish_md5.fish

```diff
@@ -1,7 +1,7 @@
 function fish_md5
     if type -q md5sum
         # GNU systems
-        echo (md5sum $argv[1] | string split ' ')[1]
+        echo (echo $argv[1] | md5sum | string split ' ')[1]
```

Comment:
I'm not entirely sure how I would make md5sum treat a string as a string and not a path.
I really only get it to work when piping it into `md5sum`.
So I should make a switch for `fish_md5`, where default passes as argument, and if switch is set, pipe it in?
But what's the equivalent for `md5`?